addons/account/wizard/account_move_reversal.py

```diff
@@ -113,9 +113,17 @@ def reverse_moves(self):
         moves = self.move_ids
 
         # Create default values.
+        bank_ids = self.env['res.partner.bank'].search([
```

Comment:
I checked if we could use `read_group` (or `_read_group` later on), it does not help much, as you would still have to transform it into a dict and can't limit the number of bank by partner.
Do use `search_fetch` in later versions, I guess?